M2/libraries/normaliz/patch-3.8.5

```diff
@@ -0,0 +1,84 @@
+From 21bc0dd240089fa45e9a0175eec30a2618b8a6a7 Mon Sep 17 00:00:00 2001
```

Comment:
I reported a bug with the use of openmp by normaliz in 2010 -- does this finally fix it?  (I just tell normaliz not to use openmp.)

Reply:
I think they have an issue open since August with this exact fix suggested by you, but they never merged it so I just made a PR for them.